open-grant-proposals/open-proposal-miner-analyzer.md

```diff
@@ -0,0 +1,173 @@
+Time: 2 weeks
+
+
+### Milestone 2
+
+Goals:
+ 1. Light daemon application that contains daemon initalization, reports uptime and disk space. App is developed with Go and contains unit tests.
+     * 1 backend developer
+     * 8 days
+     * 3200$
+ 2. Backend application initalization, API for user and node registration, uptime tracking service, disk occupancy tracking. App is developed in Node.Js with TypeScript and contains unit tests.
+    * 1 backend developer
+    * 8 days
+    * 3200$
+ 3. Web application initalization, design integration in React.
+    * 1 frontend developer
+    * 2 weeks
+    * 3200$
+     
+Cost: 9600$
+Time: 2 weeks
+
+### Milestone 3
+
+Goals:
+
+ 1. Integration in web application of user registration, reading node connection status, history of deals, calculating profits, slashing records.
+    * 2 frontend developers
+    * 2 weeks
+    * 6400$
+ 2. Email notifications
+    * 1 backend developer
+    * 3 days
+    * 1200$
+
+Cost: 7600$
+Time: 2 weeks
+
+### Milestone 4
+
+Goals:
+ 1. Production deployment
+    * 3 days
+    * 1200$
+ 3. QA of complete solution and bug fixing
+ 5. Project documentation
+
+Cost: 2500$
+Time: 2 weeks
+
+## Total Budget Requested
+
+Milestone 1 - 4000$
+Milestone 2 - 9600$
+Milestone 3 - 7600$
+Milestone 4 - 2500$
+
+Total: 23,700$
+
+
+| Role | Rate / day |
+| -------- | -------- |
+| Designer     | 300$     |
+| Lead developer     | 400$     |
+| Backend developer     | 400$     |
+| Frontend developer     | 320$     |
+
+
+Note that the whole project management everyday costs are free of charge due to the good faith of our team.
+
+
+## Maintenance and Upgrade Plans
+
+Complete project will be open source and therefore it can be self hosted too. We plan to introduce (at some point when there is enough users) a payed plug-and-play service that is hosted by us and that will allow us the funding to maintain the project. However, in the beginning, it will be provided and hosted by us free of charge for everyone.
+
+This project can be later expanded with more statistics depending on the Filecoin network protocol changes and updates. We are now building a separated daemon app but depending on the success level, we are open to integrating it with the Filecoin's node client.
+
+# Team
+
+## Team Members
+
+NodeFactory is a blockchain R&D company that develops dapps, infrastructure and tooling. We are a team of 6, mostly fullstack developers with blockchain development experience since 2017.
+
+## Team Member LinkedIn Profiles
+
+- [Belma Gutlić](https://www.linkedin.com/in/belmagutlic/)
+- [Marin Petrunić](https://www.linkedin.com/in/mpetrunic/)
+
+## Team Website
+
+[https://nodefactory.io](https://nodefactory.io)
+
+## Relevant Experience
+
+You can find some of our previous work in the [portfolio](https://www.nodefactory.io/Portfolio-NodeFactory-v3.pdf). Currently, we are also working on a similar application for Eth2 validators called ChainGuardian that has been funded by MolochDAO.
```

Comment:
I get a 404 when trying to access the portfolio. Could you provide a new link?

Reply:
Sorry about that, we are working on our website for the last few days. You can access it now using the same link.